litmus/director/TCID-DIR-OP-DELETE-EXTERNAL-CSPC-WITH-NO-VOLUME/test.yml

```diff
@@ -33,7 +33,7 @@
 
         ## Get node name
         - name: Get node name
-          shell: kubectl get bd -n openebs --no-headers | grep Unclaimed | awk '{print $2}' | tail -n 1
+          shell: kubectl get bd -n openebs -ojsonpath='{range .items[*]}{.metadata.name}{"\t"}{.metadata.labels.kubernetes\.io/hostname}{"\t"}{.status}{"\t"}{.spec.filesystem}{"\t"}{"\n"}{end}' | grep "Unclaimed" | grep -v "fsType" | awk {'print $2'} | tail -n 1
```

Comment:
How you are verifying this blockdevice you are getting here is exactly from the node obtained from above. Here you are using tail that means the last one which means it can be BD from any node